script/src/main.rs

```diff
@@ -0,0 +1,61 @@
+
+use sqlx::PgPool;
+use sqlx::Result;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    dotenv::dotenv().expect("Unable to load environment variables from .env file");
+
+    let db_url = std::env::var("DATABASE_URL").expect("Unable to read DATABASE_URL env var");
+    let pool = PgPool::connect(&db_url).await?;
+
+    // Define a struct to represent the rows returned by the query
+    #[derive(sqlx::FromRow)]
+    struct JobId {
+        id: i32,
+    }
+
+    // Define the SQL query to identify job IDs where total_processed in email_results is equal to total_records in bulk_jobs
+    let job_ids_to_delete_query = r#"
```

Comment:
We also need to check that the completed job is `x` days old. You can add a new environment variable to make this configurable. This way only one day old completed jobs will be deleted.